Database/Corrections/sodQuestFixes.lua

```diff
@@ -229,6 +229,24 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425170,
             [questKeys.zoneOrSort] = sortKeys.MAGE,
         },
+        [78132] = {
+            [questKeys.startedBy] = {nil,{409758}},
+            [questKeys.objectivesText] = {"Retrieve the Dragonslayer's Helm and bring it to Alonso in Ashenvale."},
+            [questKeys.requiredSpell] = -425444,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [78133] = {
+            [questKeys.startedBy] = {nil,{409754}},
```

Comment:
```suggestion
            [questKeys.objectives] = {nil,nil,{{209873}}},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5416/commits/b8b66e1032045f73d133207f1a9eb4ae59612d21